api/db/populate-messages.js

```diff
@@ -13,6 +17,21 @@ const createMessage = async (messageParams) => {
   });
```

Comment:
This could be changed too so only the full `messageParams` is passed to the `message` constructor.